tests/virt_autotest/login_console.pm

```diff
@@ -40,14 +40,23 @@ sub login_to_console {
     $timeout //= 5;
     $counter //= 240;
 
+    reset_consoles;
+    select_console 'sol', await_console => 0;
+
     if (check_var('PERF_KERNEL', '1') or check_var('CPU_BUGS', '1') or check_var('VT_PERF', '1')) {
-        reset_consoles;
-        select_console 'sol', await_console => 0;
-        send_key_until_needlematch(['linux-login', 'virttest-displaymanager'], 'ret', $counter, $timeout);
-        #use console based on ssh to avoid unstable ipmi
-        save_screenshot;
-        use_ssh_serial_console;
-        return;
```

Comment:
I am afraid this change will affect performance run, @jerrytt would you please help review?

Reply:
I just move this two lines to above, it should works as before.

Reply:
performance code branch should only run the line you delete(all code before return). and ignore all behind .
so this will cause some issues.

Reply:
@jerrytt update for only affect on CPU_BUGS